mailpoet/tests/acceptance/Settings/BuiltInCaptchaSubscriptionCest.php

```diff
@@ -26,17 +32,29 @@ public function _before(\AcceptanceTester $i) {
 
     $formName = 'Subscription Acceptance Test Form';
     $formFactory = new Form();
-    $form = $formFactory->withName($formName)->create();
+    $this->formId = $formFactory->withName($formName)->create()->getId();
 
     $subscriberFactory = new Subscriber();
     $subscriberFactory->withEmail($this->subscriberEmail)->withCountConfirmations(1)->create();
 
-    $i->cli(['widget', 'add', 'mailpoet_form', 'sidebar-1', '2', "--form={$form->getId()}", '--title="Subscribe to Our Newsletter"']);
+    $i->havePostInDatabase([
+      'post_author' => 1,
+      'post_type' => 'page',
+      'post_name' => 'form-test',
+      'post_title' => 'Form Test',
+      'post_content' => '
```

Comment:
The commit that added this code says '[Refactor acceptance tests](https://github.com/mailpoet/mailpoet/pull/5612/commits/d7d07385694f386024b493dfe9f8948ba6441049)'

Would you mind explaining why we are changing the form?

Reply:
Maybe I was wrong with the terminology, but I had to update `createFormAndSubscribe` helper to support testing blockbased theme. There is no widgets in blockbased themes so modifying the helper method to create a form via normal post rather then widget (I believe the widget was just randomly chosen and not on purpose) solved issue with multiple themes blockbased and non-blockbased.

Reply:
Thanks, that makes sense.